src/components/elements/input/RadioGroupInput.tsx

```diff
@@ -118,7 +119,8 @@ const RadioGroupInput = ({
           }),
           ...sx,
         }}
-        {...props}
+        aria-label={ariaLabel}
```

Comment:
same here, not sure if it's in the right place. Is it on the element with the `radiogroup` role? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role

Reply:
Yes, I think so! 
<img width="232" alt="Screenshot 2024-03-04 at 10 27 00 AM" src="https://github.com/greenriver/hmis-frontend/assets/16002775/65acf5db-496a-4e86-a674-7736f76e98b6">